test/test_data_collector.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import errno
+import os
+from pathlib import Path
+import time
+
+from ros_cross_compile.data_collector import DataCollector
+from ros_cross_compile.data_collector import DataTimer
+from ros_cross_compile.data_collector import Datum
+
+
+def test_construction():
```

Comment:
whenever possible try to have one assert per test case (i.e. better to have more test cases whenever possible)

Reply:
it's good to have explicit names for test cases. I.e. check_that_datum_constructor_does_not_throw